library/types/src/modules/String.rb

```diff
@@ -499,24 +499,17 @@ def ParseOptions(options, parameters)
                 "r"  => " ", # CR
                 "t"  => "\t", # tab
                 "v"  => "v", # vertical tab
-                "\\" => "\\"
-              } # backslash
-
-              if Builtins.haskey(backslash_seq, nextcharacter) == true
-                character = Ops.get_string(
-                  backslash_seq,
-                  nextcharacter,
-                  "DUMMY"
-                )
+                "\\" => "\\", # backslash
+                # backslash will be removed later,
+                # double quote and escaped double quote have to different yet
```

Comment:
I just copy it, to be honest I have no idea what it mean and hope there is good enough testsuite for this parser :)